spec/models/comment_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
Apparently we can add some unit tests for models as per this gem: https://github.com/thoughtbot/shoulda-matchers